tests/features/warmshowers/not-logged-in.feature

```diff
@@ -1,40 +0,0 @@
-Feature: User profile should say not available when user marks status as unavailable
```

Comment:
Can we keep this file please. I think it may need to be rewritten a little, but it is an original feature written by Kosta and a good example to follow.